storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOColumnMeta.java

```diff
@@ -41,6 +41,17 @@ public boolean isMethod() {
         return element.getKind() == ElementKind.METHOD;
     }
 
+    @NotNull
+    public String getRealElementName() {
+        if (elementName.startsWith("get") && Character.isUpperCase(elementName.charAt(3))) {
+            return elementName.substring(3).toLowerCase();
```

Comment:
Silly me, of course, you are right! I just wanted to decapitalize the first letter instead of doing full `toLowerCase`.